File: submissions/python3/769.py
class Solution:
    def maxChunksToSorted(self, arr):
        """
        :type arr: List[int]
        :rtype: int
        """
        maxChunks, til = 0, 0
        for i in range(len(arr)):
            til = max(til, arr[i])
            if til == i:
                maxChunks += 1
        return maxChunks

File: submissions/cpp/769.cpp
class Solution {
public:
    int maxChunksToSorted(vector<int> &arr) {
        int maxChunk = 0, til = 0;
        for (int i = 0; i < arr.size(); ++i) {
            til = max(til, arr[i]);
            if (til == i)
                ++maxChunk;
        }
        return maxChunk;
    }
};
